L/likwid/bundled/patches/useperfevents.patch

```diff
@@ -0,0 +1,28 @@
+diff --git before/likwid-5.2.1/config.mk after/likwid-5.2.1/config.mk
+index aa623fd..0d6e395 100644
+--- before/likwid-5.2.1/config.mk
++++ after/likwid-5.2.1/config.mk
+@@ -11,12 +11,12 @@
+ COMPILER = GCC#NO SPACE
+ 
+ # Path were to install likwid
+-PREFIX ?= /usr/local#NO SPACE
```

Comment:
There shouldn't be a need to patch anything, you can set the variables on the command line when invoking make:
```
make PREFIX=${prefix} ...
```
Also, don't hardcode absolute paths, use the relevant [environment variables](https://docs.binarybuilder.org/stable/build_tips/#Automatic-environment-variables)